src/Promitor.Docker.dcproj

```diff
@@ -10,9 +10,6 @@
     <DockerServiceName>promitor.scraper</DockerServiceName>
   </PropertyGroup>
   <ItemGroup>
-    <None Include="docker-compose.vs.debug.yml">
```

Comment:
Please don't remove this

Reply:
Or is it because it breaks your local experience? if so we should check to make it optional

Reply:
Done !

Reply:
Thanks!